regression/create_test_script.py

```diff
@@ -83,18 +83,26 @@ def joining(lst):
                 if log_data == "VGGFace2_FaceDetection":
                     continue
 
-                temp[i+1] = str(config[log_data][log_model]["epoch"])
+                try:
+                    temp[i+1] = str(config[log_data][log_model]["epoch"])
+                except KeyError:
+                    # Handle the KeyError by assigning a dummy value
+                    temp[i+1] = "10"
 
                 if '--deterministic' not in temp:
                     temp.insert(-1, '--deterministic')
 
                 temp.insert(-1, '--name ' + log_name)
 
-                data_name = temp[k+1]
-                if data_name in config and "datapath" in config[data_name]:
+                try:
                     path_data = config[log_data]["datapath"]
-                    temp.insert(-1, '--data ' + path_data)
+                    temp[i+1] = str(config[log_data][log_model]["epoch"])
+                except KeyError:
+                    # Handle the KeyError by assigning a dummy value
+                    path_data = "/data_ssd"
```

Comment:
it seems like the default path is the oath in the server. isn't it better for the script to give an exception and exit when the data path would not be given in the config file?

Reply:
Handled with a warnings message.